source/codegen/metadata/nirfmxinstr_restricted/functions.py

```diff
@@ -957,27 +894,14 @@
             },
             {
                 'direction': 'in',
-                'name': 'selectorString',
-                'type': 'char[]'
-            },
-            {
-                'direction': 'in',
-                'name': 'arraySize',
-                'type': 'int32'
-            },
-            {
-                'direction': 'out',
-                'name': 'externalAttenuationTableNames',
-                'size': {
-                    'mechanism': 'ivi-dance',
-                    'value': 'arraySize'
-                },
+                'name': 'filePath',
                 'type': 'char[]'
             }
         ],
         'returns': 'int32'
     },
-    'GetActiveTableName': {
+    'LoadConfigurationsFromJSON': {
+        'codegen_method': 'CustomCode',
```

Comment:
I don't think you need `CustomCode` at the `function` level for this. You use a `custom-code` size mechanism. [Example](https://github.com/ni/grpc-device/blob/d30a1cb0c11b9622e0c4954d3a99d755992a3dde/source/codegen/metadata/nidaqmx/functions.py#L142).

```python

            {
                'direction': 'in',
                'name': 'arraySize',
                'size': {
                    'mechanism': 'custom-code',
                    'value': 'jsonString.length()'
                },
                'type': 'int32'
            }
```

Reply:
I tried it, and it doesn't actually eliminate the arraySize parameter from the grpc request message. I think it is because the 'size' tag doesn't apply to the scalar int32 value.

Let me know if I'm not understanding this the right way around, but I don't think I can add the size tag to the jsonString parameter either to achieve this.

Reply:
I must've glossed over this the first time. I think something like this will get you the generated code you want:

```
'LoadConfigurationsFromJSON': {
        'parameters': [
            {
                'direction': 'in',
                'grpc_name': 'instrument',
                'name': 'instrumentHandle',
                'type': 'niRFmxInstrHandle'
            },
            {
                'direction': 'in',
                'name': 'jsonString',
                'type': 'char[]',
                'size': {
                    'mechanism': 'len',
                    'value': 'arraySize',
                },
            },
            {
                'direction': 'in',
                'name': 'arraySize',
                'type': 'int32'
            }
        ],
        'returns': 'int32'
    },
```

Reply:
Thanks for the example, Ryan! This did the trick.